zerver/models.py

```diff
@@ -208,7 +208,7 @@ def __unicode__(self):
 
     @cache_with_key(get_realm_emoji_cache_key, timeout=3600*24*7)
     def get_emoji(self):
-        # type: () -> Dict[Text, Optional[Dict[str, Iterable[Text]]]]
+        # type: () -> Dict[Text, Dict[str, Iterable[Text]]]
```

Comment:
Hmm, when I go consult `get_realm_emoji_uncached` to check this, taking out the `Optional` is right -- but it looks like there's a different issue, in that that return type is `Dict[Text, Dict[str, Any]]`. So effectively this annotation is making a promise that those inner values will be `Iterable[Text]`, but looking at the implementation it doesn't seem like that's true.